### PR TITLE
change default behavior so action fails on medium (and higher) severities

### PR DESCRIPTION
**Pull request by alfredodeza**
Fixes issue #83

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -15,9 +15,9 @@ inputs:
     required: false
     default: 'false'
   fail-build:
-    description: 'Set to any value to cause build to fail upon failed anchore policy evaluation'
+    description: 'Set to false to avoid failing based on severity level. Default is to fail when severity threshold is met'
```
Comment (luhring) [inline action.yml:18]: I'd suggest being consistent with what we call the severity threshold. Here was say "severity threshold", but it looks like elsewhere in this action's settings we say "severity-cutoff"

Comment (alfredodeza) [reply action.yml:18]: @luhring I went to make the change, but the thing is that `severity-cutoff` is the key, not a _wording_. So I could make everything refer to that key instead. Is that what you were thinking too? 

That is: the _severity threshold_ is the value for the `severity-cutoff` configuration key

Comment (luhring) [reply action.yml:18]: @alfredodeza I think I'm missing something - perhaps a 3 min zoom session would set me straight 😄

Comment (alfredodeza) [reply action.yml]: I've updated this to use `severity-cutoff` instead of _"severity threshold"_

**Review by luhring — APPROVED** [review]
Looks good!

**Pull request merged**